packages/ui/.env

```diff
@@ -1 +1,2 @@
 PORT=8000
+REACT_APP_EVENT_STORE_URL=http://localhost:3000
```

Comment:
You can't use `ui/.env` in production, we will only use `server/.env` when building docker image. You have to define it in `server/.env`, and export it in `[/config](https://github.com/CasperLabs/clarity/blob/49982c03a43e963a624a9bbc7e28ccb5a04e07a4/packages/server/src/server.ts#L171)` endpoint.